package.json

```diff
@@ -8,6 +8,7 @@
     "clean": "turbo run clean --force --parallel",
     "build": "turbo run build",
     "test": "turbo run test",
+    "test:integration": "turbo run test:integration",
```

Comment:
The test Weather client needs generated and built before integration tests can be run. Should this be `./gradlew clean build && turbo run test:integration` instead? If someone is working on integration tests, they may or may not need to run codegen, depending on their changes. However, if you're working on a single dependency package's integration tests, you may opt to run `yarn test-integration` from the local package instead, to avoid running tests across all of the packages.